Font style utilities did not contained "-style-" in BS4 (#37361)
twbs/bootstrap: site/content/docs/5.2/migration.md

- <span class="badge bg-danger">Breaking</span> Renamed `.font-weight-*` utilities as `.fw-*` for brevity and consistency.

> - <span class="badge bg-danger">Breaking</span> Renamed font style utilities `.font-italic` and `.font-normal` as `.fst-*` for brevity and consistency.

Reviewer: ```suggestion
- <span class="badge bg-danger">Breaking</span> Renamed `.font-italic` utility to `.fst-italic` for brevity and consistency with new `.fst-normal` utility.
```
